recipes/physx/4.1.1/CMakeLists.txt

```
@@ -0,0 +1,9 @@
+cmake_minimum_required(VERSION 2.8)
+project(cmake_wrapper)
+
+include(conanbuildinfo.cmake)
+conan_basic_setup()
+
+option(PHYSX_CONAN_FPIC ON)
```

```suggestion
option(PHYSX_CONAN_FPIC "Build physx with fPIC" ON)
```
see https://cmake.org/cmake/help/v3.13/command/option.html
(the second argument is documentation)

---

fixed